apps/site/navigation.json

```diff
@@ -323,6 +323,10 @@
             "backpressuringInStreams": {
               "link": "/learn/modules/backpressuring-in-streams",
               "label": "components.navigation.learn.modules.links.backpressuringInStreams"
+            },
+            "howToUseStreams": {
```

Comment:
This learn page should be before backpressure in streams IMO.

Reply:
I'll change it, I thought they needed to be in order of creation, but it makes sense to have this first as it is more basic/general

Reply:
Done